README.md

```diff
@@ -24,58 +23,72 @@ If you have any feature requests, feel free to raise an issue or contact us by e
     - [Plotting with GRIB/NETCDF](#plotting-gribnetcdf-icon-data)
     - [Specifying Vertical Level](#specifying-vertical-level)
-5. [FAQ](#faq)
+6. [FAQ](#faq)
     - [Trouble shooting](#trouble-shooting)
-6. [Contacts](#contacts)
-7. [Acknowledgments](#acknowledgments)
+7. [Contacts](#contacts)
+8. [Acknowledgments](#acknowledgments)
 
 
 # Getting started with psyplot
 ## Environment Setup
 
-We recommend to use a conda environment for the usage of the provided scripts. Please follow the instruction for the installation.
+We recommend to use a conda environment created with mamba for the usage of the provided scripts. Please follow the instruction for the installation.
 
-### Install Miniconda
 <details>
-	<summary> <b><u> Instructions </u></b> </summary>
+	<summary> <b><u> Instructions to install Mamba (recommended) </u></b> </summary>
+
+1. See [Mamba documentation](https://mamba.readthedocs.io/en/latest/mamba-installation.html#mamba-install) for help.
+2. Do a fresh install of mambaforge on your `$HOME` directory (default location);
+        
+       wget "https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-$(uname)-$(uname -m).sh"
+
+       bash Mambaforge-$(uname)-$(uname -m).sh
+
+3. Restart your shell to use mamba.
+4. Install all environments on your `$PROJECT` directory (Piz Daint) or on your `$SCRATCH` (Tsa), otherwise you risk filling up your $HOME directory. See below for instructions.
+
+</details>
+
+<details>
+	<summary> <b><u> Instructions to install Conda </u></b> </summary>
 
 1. Look up most recent Miniconda version for Linux 64-bit on the [Miniconda documentation pages](https://docs.conda.io/en/latest/miniconda.html)
 2. Install miniconda on your `$HOME` directory (default location);
         
        wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
 
        bash Miniconda3-latest-Linux-x86_64.sh
-
+   
 3. Install all environments on your `$PROJECT` directory (Piz Daint) or on your `$SCRATCH` (Tsa), otherwise you risk filling up your $HOME directory. See below for instructions.
 
 </details>
 
 ### Create conda environment
 In the following instructions replace $PROJECT by $SCRATCH if using Tsa (instead of Piz Daint).
 
-Create a conda environment _psyplot_ with python[version>=3.7,<3.10] (psy-view requirement) and install requirements:
+Create a conda environment _psyplot_ with python[version>=3.7,<3.10] (psy-view requirement) and install requirements (if conda takes too long to solve, try using [mamba](https://mamba.readthedocs.io/en/latest/mamba-installation.html#mamba-install) or using the pinned environment: env/environment_pinned.yml):
 
-    conda env create --prefix $PROJECT/envs/psyplot -f env/environment.yml
+    mamba env create --prefix $PROJECT/envs/psyplot -f env/environment.yml
 
 To be able to activate your conda environment by simply using `conda activate psyplot` instead of the full path, add the following to your `.bashrc`:
```

Comment:
Should we change to `mamba activate psyplot`?

Reply:
Maybe best to change it, even though i think both work

Reply:
already done!